build.gradle

```diff
@@ -1,31 +0,0 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
```

Comment:
Please revert these changes. This file should not be deleted. 
Once done, please open the `Files changed` section on your PR and make sure this file is not visible.

Reply:
> Hello @rt4914, Sir, please review it again, the error has occurred because I've not added these above files in commit mistakingly.

@TheSwarnim Thanks. Also make sure that you reply to all open comments so that the reviewer can understand what was the step taken on that particular comment.